### PR TITLE
Doc: short cluster intro / cluster of one call-out in cluster_install

### PR DESCRIPTION
**Pull request by barentine**
- write short blurb on PYMECluster doc landing page
- in cluster_install, clarify that clusterofone is already installed if that's how one wants to use it
- remove low level API heading for now since we haven't added it yet

### CODE REVIEW

**Comment by David-Baddeley** [comment]
could you roll back the low level API change - fixing up the cluster_compute docs on today's TODO list.

**Comment by David-Baddeley** [comment]
(I just pushed as is so we had new executables to test).

**Comment by barentine** [comment]
@David-Baddeley  done

File: docs/cluster/cluster_install.rst [hunk 1]
```diff
@@ -3,8 +3,34 @@ Setting up PYME cluster hardware and software
...
+
+in a terminal window (OSX, Linux) or *"Anaconda prompt"* (Windows) [#anacondaprompt]_ . 
+On Windows you may be prompted to allow python through the firewall, which you
+should accept. 
+
+A browser page for monitoring active analysis tasks will be 
+opened. Note that a more complete browser-based user interface for the cluster
+can be opened by modifying the same terminal command with the flag `--clusterUI`. 
+This improved interface allows initiation of analysis rather than just monitoring,
+however it requires the installation of the python package Django, version 2.1,
```
Comment (David-Baddeley) [inline docs/cluster/cluster_install.rst]: now in pyme-depends ... (5 months ago)

Comment (barentine) [reply docs/cluster/cluster_install.rst]: good! just adjusted that sentence

**Comment by David-Baddeley** [comment]
Copy and pasted the essentials across in #1291, but  with substantial edits (I think it's best to keep ClusterOfOne out of clusterInstall for clarity, so reduced that to a reference - basically just "if you want clusterofone you are in the wrong place here")

**Comment by David-Baddeley** [comment]
relevant bits copied to #1291

**Pull request closed without merging**